src/renderer/register/components/AccountView/ImportView/ImportView.js

```diff
@@ -0,0 +1,111 @@
+import * as bip39 from 'bip39';
+import { debounce } from 'lodash';
+import React from 'react';
+import { bool, func, arrayOf, string } from 'prop-types';
+
+import AuthPanel from 'auth/components/AuthPanel';
+import NavigationButtons from 'shared/components/NavigationButtons';
+import { DEFAULT_LANGUAGE } from 'shared/values/languages';
+import registerShape from 'register/shapes/registerShape';
+
+import MnemonicWord from './MnemonicWord';
+import styles from './ImportView.scss';
+
+export default class ImportView extends React.PureComponent {
+  static propTypes = {
+    account: registerShape.isRequired,
+    loading: bool,
+    onCancel: func.isRequired,
+    previousStep: func.isRequired,
+    storeFormData: func.isRequired,
+    showErrorToast: func.isRequired,
+    mnemonic: arrayOf(string).isRequired,
+    setMnemonic: func.isRequired
+  };
+
+  static defaultProps = {
+    loading: false
+  };
+
+  state = {
+    validMnemonic: false
+  };
+
+  mnemonicRefs = Array(24)
+    .fill(undefined)
+    .map(() => React.createRef());
+
+  showErrorToast = debounce((e) => {
```

Comment:
Why is this debounce required? I'm not sure what the resulting behavior will be

Reply:
So it doesn't pop up on every key press